index.html

```diff
@@ -4,10 +4,126 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" >
     <link rel="stylesheet" href="style.css">
     <title>Component Library Documentation</title>
 </head>
 <body>
+    <div class="container">
+        <h1>Avatar</h1>
+        <hr>
+        
+        <h2>Avatar with 6 sizes</h2>
+        <br>
+        <div class="component">
+            <div class="avatar avatar-size-xxl ">
+                <img src="./assets/photos/stock-photo.jpg" class="avatar-image" alt="">
+            </div>
+            <div class="avatar avatar-size-xl">
+                <img src="./assets/photos/stock-photo2.jpg"class="avatar-image" alt="">
+            </div>
+            <div class="avatar avatar-size-lg">
+                <img src="./assets/photos/stock-photo3.jpg"class="avatar-image" alt="">
+            </div>
+            <div class="avatar">
+                <img src="./assets/photos/stock-photo4.jpg"class="avatar-image" alt="">
+            </div>
+            <div class="avatar avatar-size-sm">
+                <img src="./assets/photos/stock-photo.jpg"class="avatar-image" alt="">
+            </div>
+            <div class="avatar avatar-size-xs">
+                <img src="./assets/photos/stock-photo2.jpg"class="avatar-image" alt="">
```

Comment:
alts are empty

Reply:
Yup anyways these documentation part but still done with changes. If everything is ok please approve.